open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/authorize/utils/ConsentRetrievalUtil.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
- *
+ * <p>
```

Comment:
Check the license header here, These `<p>` tags should not be here right?

Ref : https://github.com/wso2/financial-open-banking/blob/main/open-banking-accelerator/components/com.wso2.openbanking.accelerator.gateway/src/main/java/com/wso2/openbanking/accelerator/gateway/cache/CertificateRevocationCache.java#L1

Reply:
Fixed in [60a3cf3]